src/pages/Query.tsx

```diff
@@ -205,26 +196,22 @@ const Query = () => {
             </div>
             <div className="space-y-1">
               <Label
-                htmlFor="token"
-                tooltip="Your Github access token will be used to download the repository to know what to fetch from the SearchSECO database. Only read access is required for the access token."
+                htmlFor="branch"
+                tooltip="The branch to query the SearchSECO database for"
               >
-                Github access token
+                Branch
               </Label>
-              <ErrorWrapper name="Token" error={errors.token}>
+              <ErrorWrapper name="Branch" error={errors.branch}>
                 <Input
-                  {...register('token', {
-                    required: true,
-                    pattern: {
-                      value: /^[\w\d-]+$/, //placeholder
-                      message: 'Invalid token',
-                    },
+                  {...register('branch', {
+                    required: false,
                   })}
                   type="text"
-                  placeholder="Your Github token"
-                  id="token"
-                  aria-invalid={errors.token ? 'true' : 'false'}
-                  error={errors.token}
-                  defaultValue={storedToken}
+                  placeholder="master"
+                  id="branch"
+                  aria-invalid={errors.branch ? 'true' : 'false'}
+                  error={errors.branch}
+                  defaultValue="master"
```

Comment:
As we discussed, we probably do not want defaultValue, should be `undefined` by default